widgetry/examples/demo.rs

```diff
@@ -10,9 +10,9 @@ use rand::SeedableRng;
 use rand_xorshift::XorShiftRng;
```

Comment:
Hmmm... after rebasing, I'm getting a crash upon launching the demo. I'm getting the same from HEAD of master:

```
thread 'main' panicked at 'assertion failed: `(left == right)`
  left: `1`,
 right: `0`', /Users/mkirk/.cargo/registry/src/github.com-1ecc6299db9ec823/lyon_tessellation-0.16.0/src/fill.rs:813:9
stack backtrace:
   0: <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt
   1: core::fmt::write
   2: std::io::Write::write_fmt
```

Are you seeing this @dabreegster?

Reply:
Maybe related? 

https://github.com/nical/lyon/issues/607

I just pushed up a commit to this branch which reverts the recent lyon upgrade and now demo launches fine.